src/vim-webgl-viewer/viewer.ts

```diff
@@ -128,40 +128,44 @@ export class Viewer {
   private _updateId: number
 
   constructor (options?: PartialSettings) {
-    this.config = getConfig(options)
+    this.settings = getConfig(options)
```

Comment:
For consistency, it might be worth renaming "getConfig" to "getSettings" if the term we're using here is "settings"

Reply:
Right, thanks!